patch-package
--- a/node_modules/async/dist/async.js
+++ b/node_modules/async/dist/async.js
@@ -202,7 +202,7 @@ function asyncify(func) {
             result.then(function(value) {
                 invokeCallback(callback, null, value);
             }, function(err) {
-                invokeCallback(callback, err.message ? err : new Error(err));
+                invokeCallback(callback, err && err.message ? err : new Error(err));
             });
         } else {
             callback(null, result);
